frontend/src/tabs/voting/Voting.js

```diff
@@ -3,18 +3,35 @@ import BasicTab from '../BasicTab.js';
 import './Voting.css';
 import VoteItem from './VoteItem.js';
 import {
+    fetchWrapper,
   getVotableColors,
   voteColorDevnet
 } from '../../services/apiService.js';
 
 const Voting = (props) => {
   const [userVote, setUserVote] = useState(-1);
+  const [address, _setAddress] = useState(
```

Comment:
The address can be passed as a prop from `src/App.js`, that is where it is being imported for other queries as well.